### ISSUE TITLE
'save_to_file' produces a corrupt mp3 file

### ISSUE THREAD

**Issue by rubenmedrano** [nateshmbhat/pyttsx3#171]
'save_to_file' creates a corrupt mp3 file.

I have tested the result mp3 with foobar2000, fre:ac, and Ableton Live, and none of them could read it.
Only windows media player reads it.

I am using Windows 10 / Python 3.8 64bit

import pyttsx3
engine = pyttsx3.init()
engine.save_to_file('hola amigo', 'test.mp3')
engine.runAndWait()

**Comment by rubenmedrano** [comment]
OK, I have found that it is not creating a mp3 file actually, but a WAV file.

I have inserted a new line after the line 67 in the 'sapy5.py' file:

stream.Format.Type = 34

It allows to save in 44.1khz 16bit mono WAV file.

And then use, in my program:

engine.save_to_file('hola amigo', 'test.wav')

so it creates a correct WAV file.

The list of the WAV formats is here:

[https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ms720595(v=vs.85)?redirectedfrom=MSDN
](url)
